packages/eslint-config-finn/index.js

```diff
@@ -15,6 +15,7 @@ module.exports = {
         'comma-dangle': ['error', 'always-multiline'],
         'computed-property-spacing': 'error',
         'eqeqeq': ['error', 'smart'],
+        'lines-around-directive': ['error', { before: 'never', after: 'always' }],
```

Comment:
A bunch of examples in the eslint docs.
But then they'd have to be together. Do we use header comments though?